app/src/pages/playground/PlaygroundChatTemplate.tsx

```diff
@@ -37,7 +37,23 @@ export function PlaygroundChatTemplate(props: PlaygroundChatTemplateProps) {
           <li key={index}>
             <Card
               title={
-                <MessageRolePicker includeLabel={false} role={message.role} />
+                <MessageRolePicker
+                  includeLabel={false}
+                  role={message.role}
+                  onChange={(role) => {
+                    updateInstance({
+                      instanceId: id,
+                      patch: {
+                        template: {
+                          __type: "chat",
+                          messages: template.messages.map((message, i) =>
+                            i === index ? { ...message, role } : message
```

Comment:
non blocking: I'm thinking we need a rudamentary ID system here so drag and drop works. Let's to as a follow-up